src/app/modals/login-modal/login-modal.component.scss

```diff
@@ -0,0 +1,37 @@
+/*
+Copyright 2020-2021 University of Oxford
+and Health and Social Care Information Centre, also known as NHS Digital
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+.mdm-login-modal {
+
+  mat-divider {
+    margin: 4px 0;
+  }
+
+  &__providers {
+
+    &--scrollable {
+      max-height: 200px;
```

Comment:
i'd recommend making this proportion of the screen height this would allow smaller screens to show without issue

Reply:
Agreed, tried to make adjustments.